cmd/protoc-gen-openapi/generator/openapi-v3.go

```diff
@@ -470,6 +482,7 @@ func (g *OpenAPIv3Generator) buildOperationV3(
 	file *protogen.File,
 	operationID string,
 	tagName string,
+	methodName string,
 	description string,
 	defaultHost string,
 	path string,
```

Comment:
not required of course, but you can clean up a longer list of params like this by consolidating to: 
`operationID, tagName, methodName, description, defaultHost, path string`